README.md

```diff
@@ -1,521 +1,521 @@
+
+```javascript
+const middy = require('middy')
+const myMiddleware = require('myMiddleware')
+
+const handler = middy((event, context, callback) => {
+  // do stuff
+})
+
+handler.use(myMiddleware({
+  option1: 'foo',
+  option2: 'bar'
+}))
+
+module.exports = { handler }
+```
+
+
+### Inline middlewares
+
+Sometimes you want to create handlers that serve very small needs and that are not
+necessarily re-usable. In such cases you probably will need to hook only into one of
+the different phases (`before`, `after` or `onError`).
+
+In these cases you can use **inline middlewares** which are shortcut function to hook
+logic into Middy's control flow.
+
+Let's see how inline middlewares work with a simple example:
+
+```javascript
+const middy = require('middy')
+
+const handler = middy((event, context, callback) => {
+  // do stuff
+})
+
+handler.before((handler, next) => {
+  // do something in the before phase
+  next()
+})
+
+handler.after((handler, next) => {
+  // do something in the after phase
+  next()
+})
+
+handler.onError((handler, next) => {
+  // do something in the on error phase
+  next()
+})
+
+module.exports = { handler }
+```
+
+As you can see above, a middy instance exposes also the `before`, `after` and `onError`
+methods to allow you to quickly hook-in simple inline middlewares.
+
+
+### More details on creating middlewares
+
+Check the [code for existing middlewares](/src/middlewares) to have more examples
+on how to write a middleware.
+
+
+## Available middlewares
+
+Currently available middlewares:
+
+ - [`cache`](/docs/middlewares.md#cache): A simple but flexible caching layer
+ - [`cors`](/docs/middlewares.md#cors): Sets CORS headers on response
+ - [`doNotWaitForEmptyEventLoop`](/docs/middlewares.md#donotwaitforemptyeventloop): Sets callbackWaitsForEmptyEventLoop property to false
+ - [`httpContentNegotiation`](/docs/middlewares.md#httpcontentnegotiation): Parses `Accept-*` headers and provides utilities for content negotiation (charset, encoding, language and media type) for HTTP requests
+ - [`httpErrorHandler`](/docs/middlewares.md#httperrorhandler): Creates a proper HTTP response for errors that are created with the [http-errors](https://www.npmjs.com/package/http-errors) module and represents proper HTTP errors.
+ - [`httpEventNormalizer`](/docs/middlewares.md#httpEventNormalizer): Normalizes HTTP events by adding an empty object for `queryStringParameters` and `pathParameters` if they are missing.
+ - [`httpHeaderNormalizer`](/docs/middlewares.md#httpheadernormalizer): Normalizes HTTP header names to their canonical format
+ - [`httpPartialResponse`](/docs/middlewares.md#httppartialresponse): Filter response objects attributes based on query string parameters.
+ - [`jsonBodyParser`](/docs/middlewares.md#jsonbodyparser): automatically parses HTTP requests with JSON body and converts the body into an object. Also handles gracefully broken JSON if used in combination of

```

Comment:
Is there a reason that the `automatically` is lowercased?

Reply:
Definitely not one that I know of! I'll update it.